src/pages/ManageStaff/ManageStaff.jsx

```diff
@@ -36,6 +37,9 @@ const ManageStaff = ({ cookies }) => {
   const [driverUsers, setDriverUsers] = useState([]);
   const [isSuperAdmin, setIsSuperAdmin] = useState(false);
   const [currFilter, setCurrFilter] = useState('all');
+  const [count, setCount] = useState(1);
+  const [currentUserRole, setCurrentUserRole] = useState('');
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'currentUserRole' is assigned a value but never used.